### PR TITLE
[4.0] Fix site title

### PR DESCRIPTION
**Pull request by lufog**
Fix site title
before
before
after
after

### CODE REVIEW

**Comment by ghost** [comment]
changed Title meaningful.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17831">issues.joomla.org/tracker/joomla-cms/17831</a>.</sub>

File: templates/aurora/css/template.css [hunk 1]
```diff
@@ -8199,9 +8199,10 @@ legend {
     position: relative;
     display: inline-block;
     padding: 0;
-    margin-right: 3rem;
-    font-size: 2rem;
-    color: #fefefe; }
+    margin-right: 3rem; }
+    .hgeader .navbar-brand a {
```
Comment (ciar4n) [inline templates/aurora/css/template.css]: `.hgeader`? I'm guessing this should be `.header`?

**Comment by ciar4n** [comment]
@DebuggeR94 You appear to have some differences between the SCSS and CSS. Have you edited the SCSS and then compiled to CSS or simply edited both manually?

**Comment by C-Lodder** [comment]
Please do not try and manually edit the`template.css`. It will get automatically generated when you run:
`grunt compile:site`

**Comment by lufog** [comment]
@ciar4n Yes, I fixed the files straight through the Web interface, and made a typo. In the future, I will do as it should be :)

**Comment by C-Lodder** [comment]
@DebuggeR94 - Please fix the error reported by Hound too

**Comment by lufog** [comment]
Sorry, I'm still not quite clear with the github interface, and instead of create new pull request I sent another fix here.
Unfortunately I do not know how to remove it.

**Comment by wojsmol** [comment]
@DebuggeR94 Just do new commit and revert https://github.com/joomla/joomla-cms/pull/17831/commits/9cee0d1c9251266d4c517a8086fee2803768081d.

**Comment by Quy** [comment]
I assume this is no longer applicable.

**Comment by Quy** [comment]
@franz-wohlkoenig Close since this is for the no longer available aurora template?

**Comment by ghost** [comment]
closed as stated above.

thanks for Hint @Quy.

**Pull request closed without merging**